mergify/tests/conftest.py

```diff
@@ -0,0 +1,46 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import json
+import os
+
+import mock
+import pytest
+
+if "MERGIFY_TOKEN" in os.environ:
+
+    @pytest.fixture(scope="session")
+    def dd_environment():
+        yield {
+            "mergify_api_url": "https://api.mergify.com",
+            "token": os.environ["MERGIFY_TOKEN"],
+            "repositories": {
+                os.getenv("MERGIFY_REPOSITORY", "Mergifyio/mergify"): [os.getenv("MERGIFY_BRANCH", "main")]
+            },
+        }
+
+
+@pytest.fixture
+def instance():
+    return {
+        "mergify_api_url": "https://mergify.example.com",
+        "repositories": {"owner/repository": ["main"]},
+        "token": "a_token",
+        "tags": ["test:mergify"],
+    }
+
+
+@pytest.fixture()
+def mergify_api_queues():
```

Comment:
Not a blocker, but we also have a fixture for this, [here's an example of how to use it](https://github.com/DataDog/integrations-core/blob/a8c41fb2312bc17c2e1255d3e23d3a92776cea2c/rabbitmq/tests/test_openmetrics.py#L57).